### ISSUE TITLE
Cross FK -> M2M prefetch raised UnsealedAttributeAccess in get_queryset() method.

### ISSUE THREAD

**Issue by bryanhelmig** [charettes/django-seal#26]
For example:

```python
instance = SeaLion.objects.select_related(
    'location',
).prefetch_related(
    'location__climates',
).seal().get()

instance.location.climates.all()  # boom!
```

Where `location` is a FK and `location__climates` is a M2M.

**Comment by bryanhelmig** [comment]
I'd imagine other nested related fetches also exhibit this behavior.

**Comment by charettes** [comment]
Fixed in #27.
